prow/cmd/deck/main_test.go

```diff
@@ -416,3 +419,145 @@ func TestHelp(t *testing.T) {
 	handleAndCheck()
 	handleAndCheck()
 }
+
+func TestListProwJobs(t *testing.T) {
+	templateJob := &prowapi.ProwJob{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "first",
+			Namespace: "prowjobs",
+			Labels: map[string]string{
+				"foo": "bar",
+			},
+		},
+		Spec: prowapi.ProwJobSpec{
+			Refs: &prowapi.Refs{
```

Comment:
IMO the default case should have no refs, and test that need to validate refs should set them. This is especially true since whether or not a job has refs is something we're explicitly testing in this method.